okapi-common/src/test/java/org/folio/okapi/common/OkapiClientTest.java

```diff
@@ -360,7 +360,7 @@ public void testLegacyPostOk(TestContext context) {
     client.request(HttpMethod.POST, PORT, LOCALHOST, URL + "/test1")
         .onComplete(context.asyncAssertSuccess(request -> {
           request.end();
-          request.onComplete(context.asyncAssertSuccess(response -> {
+          request.response(context.asyncAssertSuccess(response -> {
             context.assertEquals(200, response.statusCode());
           }));
         }));
```

Comment:
This is ok.
Replacing HttpClient by WebClient allows for more concise code and would have avoided this code change.
You don't need to replace HttpClient by WebClient before merging this pull request.
```
    context.assertNotNull(server);
    WebClient.create(vertx)
        .post(PORT, LOCALHOST, URL + "/test1")
        .expect(ResponsePredicate.SC_OK)
        .send()
        .onComplete(context.asyncAssertSuccess());
```